bin/p2-watch/main.go

```diff
@@ -43,6 +50,13 @@ func main() {
 	}
 	if *podClusters {
 		watchPodClusters(client, applicator)
+	} else if *watchHealthF {
+		if *healthService == "" {
+			log.Fatal("Refusing to watch entire health tree, please set a pod ID with --health-pod")
+		}
+
+		watchHealth(*healthService, client)
+		return
```

Comment:
I put this here because I don't want behaviour to change if somebody adds code below the close of the else clause.